modules/otel-collector-components.adoc

```diff
@@ -82,6 +82,55 @@ The Jaeger receiver ingests traces in the Jaeger formats.
 <4> The Jaeger Thrift Binary endpoint. If omitted, the default `+0.0.0.0:6832+` is used.
 <5> The  server-side TLS configuration. See the OTLP receiver configuration section for more details.
 
+[id="kubeletstats-receiver_{context}"]
+=== Kubelet Stats Receiver
+
+:FeatureName: The Kubelet Stats Receiver
+include::snippets/technology-preview.adoc[]
+
+The Kubelet Stats Receiver extracts metrics related to nodes, pods, containers, and volumes from the kubelet's API server. These metrics are then channeled through the metrics-processing pipeline for additional analysis.
+
+.OpenTelemetry Collector custom resource with an enabled Kubelet Stats Receiver
+[source,yaml]
+----
+config: |
+  receivers:
+    kubeletstats:
+      collection_interval: 20s
+      auth_type: "serviceAccount"
+      endpoint: "https://${env:K8S_NODE_NAME}:10250"
+      insecure_skip_verify: true
+  service:
+    pipelines:
+      metrics:
+        receivers: [kubeletstats]
+env:
+  - name: K8S_NODE_NAME # <1>
+    valueFrom:
+      fieldRef:
+        fieldPath: spec.nodeName
+----
+<1> Sets the `K8S_NODE_NAME` to authenticate to the API.
+
+The Kubelet Stats Receiver requires additional permissions for the service account used for running the OpenTelemetry Collector.
+
+.Permissions required by the service account
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
The nodes/stats need to have get,watch, list verbs. 

```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: otel-collector
rules:
  - apiGroups: ['']
    resources: ['nodes/stats']
    verbs: ['get', 'watch', 'list']
  - apiGroups: [""]
    resources: ["nodes/proxy"]
    verbs: ["get"]
```

Reply:
Apart from this rest LGTM.

Reply:
Thank you. Updated.